client/app/src/screens/Synesthesia.js

```diff
@@ -60,15 +59,7 @@ export default class SynesthsiaScreen extends Component {
       });
       return;
     }
-    InteractionManager.runAfterInteractions(() =>
```

Comment:
For the concert app to stay operational, this deleted functionality now needs to be passed in as the onEnd() function here: https://github.com/sjawhar/enchanted-brain/blob/c4bab360d6178de6b03c29ff78a82054bfe44ec0/client/app/src/App.js#L176-L184

Reply:
I changed "this.props.navigation.navigate" to "NavigationService.navigate" to use that code in App.js. Let me know if that's incorrect/ will cause issues.